src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-ok.html

```diff
@@ -0,0 +1,14 @@
+<p>
+  You can customize the text for the proceed button to better match the context
+  of the prompt you're asking your users about.
+</p>
+
+<p>Usage: <code>input message: 'Do you approve?' ok: "Yes"</code></p>
```

Comment:
Please make sure examples are tested!

```suggestion
<p>Usage: <code>input message: 'Do you approve?', ok: "Yes"</code></p>
```